compiler/testData/codegen/bytecodeText/kt43049.kt

```diff
@@ -0,0 +1,6 @@
+// TARGET_BACKEND: JVM_IR
+
+val map: Map<String, String> = hashMapOf("a" to "all", "b" to "bar", "c" to "code")
+val d: String? by map
+
+// 1 @Lorg/jetbrains/annotations/NotNull;()
```

Comment:
We use codegen/bytecodeListing tests for such cases, it makes ABI comparison between old and new back-end easier.